server/app.js

```diff
@@ -3,14 +3,51 @@ const path = require('path');
 const logger = require('morgan');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
-
+const bcrypt = require('bcrypt');
+const passport = require("passport");
+const { Strategy } = require("passport-local");
+const session = require("express-session");
 const index = require('./routes/index');
-
+const db = require('./db');
 const app = express();
 
+//login
+passport.use(
+  new Strategy(async (account, password, done) => {
+    try {
+      const user = await db.users.findByAccount(account);
+      if (!user) {
+        return done(null, false, { message: 'Incorrect account.'});
+      }
+
+      const correctPassword = await bcrypt.compare(password, user.password);
+
+      if (!correctPassword) {
+        return done(null, false, { message: 'Incorrect password.'});
+      }
+  
+      done(null, user);
+    } catch (error) {
+      done(error);
+    }
+  })
+);
+
+passport.serializeUser((user, done) => {
+  done(null, user.id);
+});
+
+passport.deserializeUser((id, done) => {
+  db
+    .users
+    .findById(id)
+    .then(user => done(null, user))
+    .catch(err => done(err));
+});
+
 // view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'jade');
+// app.set('views', path.join(__dirname, 'views'));
+// app.set('view engine', 'jade');
```

Comment:
did you need this or can you delete this 2 lines?

Reply:
Actually I want to delete these lines.